tests/e2e/yamcs/realtimeData.e2e.spec.mjs

```diff
@@ -249,6 +251,101 @@ test.describe('Realtime telemetry displays', () => {
+                const messageEvents = [];
+
+                for (let messageCount = 0; messageCount < COUNT; messageCount++) {
+                    const message = {
+                        "type": "parameters",
+                        "call": call,
+                        "seq": messageCount,
+                        "data": {
+                            "@type": "/yamcs.protobuf.processing.SubscribeParametersData",
+                            "values": [
+                                {
+                                    "rawValue": {
+                                        "type": "FLOAT",
+                                        "floatValue": 10.204108
+                                    },
+                                    "engValue": {
+                                        "type": "FLOAT",
+                                        "floatValue": 10.204108
+                                    },
+                                    "acquisitionTime": new Date(Date.now() + messageCount).toISOString(),
+                                    "generationTime": new Date(Date.now() + messageCount).toISOString(),
+                                    "acquisitionStatus": "ACQUIRED",
+                                    "numericId": 1
+                                }
+                            ]
+                        }
+                    };
+                    const event = new Event('message');
+                    event.data = JSON.stringify(message);
+                    messageEvents.push(event);
+                }
+
+                messageEvents.forEach(event => {
+                    self.currentWebSocket.dispatchEvent(event);
+                });
+
+            }, {
+                call: callNumber,
+                COUNT: PARAMETER_VALUES_COUNT
+            });
+
+            // Subscribe to Battery1_Temp so we can confirm that the injected parameter values were received,
+            const telemetryValues = await page.evaluate(async () => {
+                const openmct = window.openmct;
+                const objectIdentifier = {
+                    namespace: 'taxonomy',
+                    key: '~myproject~Battery1_Temp'
+                };
+                const telemetryObject = await openmct.objects.get(objectIdentifier);
+
+                return new Promise((resolveWithTelemetry) => {
+                    openmct.telemetry.subscribe(telemetryObject, (telemetry) => {
+                        resolveWithTelemetry(telemetry);
+                    }, {strategy: 'batch'});
+                });
+            });
+
+            expect(telemetryValues.length).toBe(PARAMETER_VALUES_COUNT);
+
+            const notification = page.getByRole('alert');
+            const count = await notification.count();
+
+            if (count > 0) {
+                const text = await notification.innerText();
+                expect(text).not.toBe('Telemetry dropped due to client rate limiting.');
```

Comment:
Would it be beneficial to have added the performanceIndicator plugin and read the value here?

Reply:
The original user report calls out this notification specifically, so I think it's a reasonable test to validate we have fixed the issue as reported.

[I have added a test to the open source repository to test the new performance overlay though.](https://github.com/nasa/openmct/pull/7837/files#diff-71cc5c9ef53eb1f19f8970f4cac9dbd45efaa6db2e2d5ffa3715936e064c39eaR46)